README.rst

```diff
@@ -1579,6 +1580,29 @@ Client automatically follows HTTP redirects. The default redirect policy is to f
 
 .. marker-asyncio-end
 
+SQL Support
+^^^^^^^^^^^
+.. marker-sql-support-start
+
+The ability to query InfluxDB with SQL was introduced with the IOX backend.
+To make use of the SQL support users can make use of the SQL Query API:
```

Comment:
Isn't the official naming ÌOx`? Just asking in case it matters... :-)

Reply:
Good question, asked and folks said avoid iox

Reply:
`was recently introduced` might also work. Comment still open.

Reply:
how about:

```
InfluxDB Cloud now has the ability to query data with SQL. To make use
of the SQL support users can create a SQL Client with this library:
```

and leave the time element of the statement out?